spec/ruby-stackoverflow/response_data_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+require 'json'
+
+module RubyStackoverflow
+  describe Client::ResponseData do
+    it 'should response data fields' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.